Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -15,26 +16,47 @@ class BRBViewController: UIViewController {
         case account(BRBAccount)
     }
     
-    private var connectionHandler: BRBConnectionHandler!
+    var accountManager: BRBAccountManager!
     private var requestStart: Date?
+    private var account: BRBAccount?
+    private var loggedIn = false
```

Comment:
Is it possible to have this be a computed property using `accountManager.stage`?

Reply:
It is, but it introduces some issues when the user is already logged in and we query their cached account. Basically, with how the current enum Stages works, they would not be able to log out until the query is completed (if it ever is).

Reply:
What about making it be based off of `self.state`? Is that possible? 

If it's not, I think it would be best to clarify the relationship between `isLoading`, `state`, and `activityIndicator`